package.json

```
@@ -30,7 +30,8 @@
 		"Amish Shah <amishshah.2k@gmail.com>",
 		"Vlad Frangu <kingdgrizzle@gmail.com>",
 		"SpaceEEC <spaceeec@yahoo.com>",
-		"Aura Román <kyradiscord@gmail.com>"
+		"Aura Román <kyradiscord@gmail.com>",
+		"Vinayak Kulkarni <inbox.vinayak@gmail.com>"
```

this section is for the core maintainers of the project.

```suggestion
		"Aura Román <kyradiscord@gmail.com>"
```

---

Cool, will remove it 😃

---

- [x] Done in 199b242